src/modules/card.marko

```diff
@@ -0,0 +1,378 @@
+<div id="card">
```

Comment:
This is good, but might be too generic. I think we do have an actual card module coming up. 
We can call it file-upload-card maybe? 
If anyone has any suggestions on this, feel free to add